### DISCUSSION TITLE
Is it possible to limit an user to only manage its own assets

### DISCUSSION THREAD

**Discussion by joeylu** [requarks/wiki#7282] [Help / Questions]
For instance, every user in a group that has permission to create/edit/delete asset, but instead each user can delete/edit other's uploads, can I limit these actions based on each user? tks

**Comment by joeylu** [comment]
or at least that the admin can see each uploaded asset's owner, history or something to keep on track?

**Reply by imysl0** [reply]
> or at least that the admin can see each uploaded asset's owner, history or something to keep on track?

Wiki.js can be synchronized with Git, so that in the Git repository, you can see who made the changes, including changes to assets.
